Src/libCZI/SingleChannelScalingTileAccessor.cpp

```diff
@@ -218,11 +218,16 @@ std::vector<int> CSingleChannelScalingTileAccessor::CreateSortByZoom(const std::
                 {
                     return true;
                 }
+                else if (!sb1.logicalRect.IsValid() ||
+                           sb2.logicalRect.IsValid())
+                {
+                    return false;
+                }
```

Comment:
Why? I am not sure, or I'd think that we should require that we have a valid `logicalRect` here.